protocol/x/clob/rate_limit/order_rate_limiter.go

```diff
@@ -193,12 +142,34 @@ func (r *cancelOrderRateLimiter) RateLimit(ctx sdk.Context, msg *types.MsgCancel
 	return err
 }
 
-func (r *cancelOrderRateLimiter) PruneRateLimits(ctx sdk.Context) {
+func (r *placeAndCancelOrderRateLimiter) RateLimitBatchCancelOrder(ctx sdk.Context, msg clobtypes.MsgBatchCancel) (err error) {
+	lib.AssertCheckTxMode(ctx)
+
+	// calcualate how mcuh each batch cancel should be weighted. Use 2 for now.
+	weight := uint32(2)
```

Comment:
TODO: check the ratio of how much work it takes to gossip/deserialize/sigverify messages compared to actually processing messages and then set this to be `X + YperId so like ceil(1+ 0.1 per ID)`

Reply:
maybe attach a ticket?

Reply:
done